Get-help.md

```diff
@@ -48,13 +43,45 @@ If you have a question about a pull request or issue, you can also reach out to
 
 You can even mention whole teams of people! For example, if you find an issue that is destabilizing the project, you could notify all the core maintainers by including `@oppia/core-maintainers` in your issue. The teams are all listed [here](https://github.com/orgs/oppia/teams).
 
+### Email
+
+We have several mailing lists in the form of Google Groups that you can join, but please note that these are less active than GitHub Discussions:
+
+* [oppia-announce](https://groups.google.com/forum/#!forum/oppia-announce) is for announcements of new releases or blog posts. It's not for asking questions though.
+* [oppia-dev](https://groups.google.com/forum/#!forum/oppia-dev) is the main mailing list for communication between developers and for technical questions. You can post to it even if you're not a member of the group. This is where you can ask questions, solicit feedback, or make developer-specific announcements (e.g. a temporary GitHub outage).
+
+You can also email your mentor with any questions. If you don't have a mentor, complete the steps on the [[wiki page for contributing code to Oppia|Contributing-code-to-Oppia]] and you'll be assigned one.
+
 ## How to Ask Good Questions
 
-At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us with enough information to help us resolve it faster. We've divided the questions into 2 categories - Setup-related and General questions. You can start following the sections below to understand how you can ask each of them.
+At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us with enough information to help us resolve it faster. 
+
+### General guidelines to follow 
+
+* __Be clear and concise__: Clearly articulate your question to avoid confusion and allow others to understand it easily.
+* __Provide context__: If applicable, provide relevant background information to help others understand your question and situation.
+* __Be respectful__: Maintain a respectful and courteous tone in your communication to foster a positive and inclusive community environment.
+* __Use proper formatting__: Organize your question using paragraphs, bullet points, and appropriate formatting to enhance readability.
+* __Proofread your message__: Take a moment to proofread your question to ensure clarity and accuracy before sending it.
+
+### Providing error information 
+
+When reporting an error, follow these steps to communicate the issue effectively:
+
+* __Start with the command__: Rather than saying "I have an error," provide the actual command you ran. Copy and paste the command in your email or chat message to give recipients the complete context.
+* __Include the specific error message__: Share the exact error message you encountered. This helps others quickly identify the potential cause and provide relevant solutions or suggestions.
```

Comment:
Maybe add another bullet point about including relevant log lines in addition to the error message. Sometimes it's helpful to include log lines from the browser console or from the terminal if they lend helpful context. For example, if there's an error when installing Oppia, the log lines around the error can tell us how far the installation got before failing. I's also important to only include relevant lines and cut out irrelevant stuff to keep the report readable.